### DISCUSSION TITLE
Virtual Desktop , Steam : Insurgency Sandstorm

### DISCUSSION THREAD

**Discussion by GitHublovin** [praydog/REFramework#520] [General Questions] [answered]
Hi, does reframework works on "Virtual Desktop" ? I tried unpacking it in the Steam\steamapps\common\sandstorm map but i am clueless to go from this point. Will there be a instruction youtube on this ?

**Comment by praydog** [comment, marked as answer]
This project isn't for the Unreal Engine, it's for the RE Engine. The Unreal injector isn't released yet.
